core/src/epicli/data/common/ansible/playbooks/kubernetes_node.yml

```diff
@@ -3,6 +3,18 @@
   gather_facts: true
   tasks: []
 
+- hosts: kubernetes_master
+  become: true
+  become_method: sudo
+  pre_tasks:
+    - import_role:
+        name: kubernetes_common
+        tasks_from: gather-facts
+  vars_files:
+    - roles/kubernetes_master/vars/main.yml
+  environment:
+    KUBECONFIG: "{{ kubeconfig.remote }}"
+
```

Comment:
Why we need to add the common role on kubenetes_master hosts in this playbook? This task is already defined in kubernetes_master playbook so what is the case (just curious :) )?

Reply:
This is kinda weird. It is a bug. In 2.8 it was ignored for some reason, but 2.10 throws fatal error later, because of undefined "kubernetes_common" fact. So this is needed here to populate facts on kubernetes_master inventory hosts inside the hostvars dictionary.

Reply:
Basically without it "kubernetes_node" provisioning fails. :slightly_frowning_face:

Reply:
Is it required to run for all hosts in `kubernetes_master` group?

Reply:
Yes I believe so, because it tries to determine if masters are actually a part of the cluster and we cannot assume that any of them is working correctly. We have to understand it. :)